spec/tags/whatsapp.md

```diff
@@ -34,3 +34,26 @@ When you receive a message from one contact, the sender and recipient is inverte
 * Sender: is the phone number of contact
 
 In API the sender is the field `from` and the receiver is the field `to` of message object.
+
+
+## Type of Messages
+
+* **Template Message**
+
+**What's the template message?**
+
+Message Templates are message formats for common reusable messages a business may want to send. Businesses must use Message Templates for sending notifications to customers.
+
+This allows a business to send just the template identifier along with the appropriate parameters instead of the full message content.
+
+References: <a href="https://developers.facebook.com/docs/whatsapp/message-templates/" target="_blank">WhatsApp Business API</a>
+
+**How can I purchase a new template of WhatsApp?**
+
+If you already have a WhatsApp business account just send email to *whatsapp@zenvia.com*
+
+* **Session Message**
+
+**What's session message and how it works?**
+
+Whenever the end user responds a 24 hour session window is opened within whatsapp, any message trafficked within this period is considered a session message, so any kind of message (text, photo, video) are considered session message within the window. The 24-hour window only opens when the end user contacts you. Only messages trafficked (sent and received) will be billed.
```

Comment:
Actually, the window is based on the last message from the user. So whenever the user sends a new message, the window is reset.